sdk/src/Services/DynamoDBv2/Custom/DataModel/TransactWrite.cs

```diff
@@ -437,6 +435,43 @@ private Expression CreateConditionExpressionForVersion(ItemStorage storage)
                 DocumentTransaction.TargetTable.IsEmptyStringValueEnabled);
             return DynamoDBContext.CreateConditionExpressionForVersion(storage, conversionConfig);
         }
+        
+
+        private void AddDocumentTransaction(ItemStorage storage, Expression conditionExpression)
+        {
+            var hashKeyPropertyNames = storage.Config.HashKeyPropertyNames;
+            var rangeKeyPropertyNames = storage.Config.RangeKeyPropertyNames;
+
+            var keyList = storage.Document.Keys.ToList();
+
+            foreach (var keyPropertyName in hashKeyPropertyNames)
+            {
+                keyList.Remove(keyPropertyName);
+            }
+
+            foreach (var rangeKeyPropertyName in rangeKeyPropertyNames)
+            {
+                keyList.Remove(rangeKeyPropertyName);
+            }
+
+            if (keyList.Any())
```

Comment:
Add comment that if there is nothing in the `attributeNames` collection that means the user is attempting to save a POCO that only has DDB keys defined and you can't not do an update call when there are no data attributes.